src/main/java/gift/service/AuthService.java

```diff
@@ -0,0 +1,33 @@
+package gift.service;
+
+import gift.dto.AuthResponseDto;
+import gift.dto.MemberRequestDto;
+import gift.util.JwtTokenProvider;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AuthService {
+
+    private final MemberService memberService;
+    private final JwtTokenProvider jwtTokenProvider;
+
+    public AuthService(MemberService memberService, JwtTokenProvider jwtTokenProvider) {
+        this.memberService = memberService;
+        this.jwtTokenProvider = jwtTokenProvider;
+    }
+
+    public AuthResponseDto register(MemberRequestDto memberRequestDto) {
+        return new AuthResponseDto(
+                jwtTokenProvider.makeJwtToken(
+                        memberService.creatMember(memberRequestDto)
+                ));
```

Comment:
중첩된 호출은 가독성 및 디버깅이 어려울 것 같은데 아래처럼 리팩토링하는 것은 어떠신가요? 
```suggestion
        Member member = memberService.createMember(memberRequestDto);
        String token = jwtTokenProvider.makeJwtToken(member);
        return new AuthResponseDto(token);
```

Reply:
넵 수정하겠습니다!